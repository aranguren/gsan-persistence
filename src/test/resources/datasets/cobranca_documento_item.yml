faturamento.conta_geral:
  - cnta_id: 1
  - cnta_id: 2
  - cnta_id: 3
  - cnta_id: 4
  - cnta_id: 5

cobranca.cobranca_documento_item:
  - cdit_id: 1
    cbdo_id: 1
    cnta_id: 1
  - cdit_id: 2
    cbdo_id: 1
    cnta_id: 1
  - cdit_id: 3
    cbdo_id: 1
    cnta_id: 1
  - cdit_id: 4
    cbdo_id: 1
    cnta_id: 1
  - cdit_id: 5
    cbdo_id: 2
    cnta_id: 2
  - cdit_id: 6
    cbdo_id: 3
    cnta_id: 3
  - cdit_id: 7
    cbdo_id: 4
    cnta_id: 4
  - cdit_id: 8
    cbdo_id: 5
    cnta_id: 5
  - cdit_id: 9
    cbdo_id: 5
    cnta_id: 5

cobranca.cobranca_documento:
  - cbdo_id: 1
  - cbdo_id: 2
  - cbdo_id: 3
  - cbdo_id: 4
  - cbdo_id: 5

faturamento.conta:
  - cnta_id: 1
    dcst_idatual: 0