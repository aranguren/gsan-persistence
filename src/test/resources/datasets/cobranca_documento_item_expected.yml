faturamento.conta_geral:
  - cnta_id: 1
  - cnta_id: 2
  - cnta_id: 3
  - cnta_id: 4
  - cnta_id: 5

cobranca.cobranca_documento_item:
  - cdit_id: 5
    cnta_id: 2
  - cdit_id: 6
    cnta_id: 3
  - cdit_id: 8
    cnta_id: 5
  - cdit_id: 9
    cnta_id: 5